tools/enumerate-devices/rs-enumerate-devices.cpp

```diff
@@ -230,6 +230,11 @@ int main(int argc, char** argv) try
     if (!playback_dev_file.empty())
         d = ctx.load_device(playback_dev_file.data());
 
+#ifdef BUILD_WITH_DDS
```

Comment:
Maybe it's better to add this code into context.cpp query_devices until the adjustment to DDS will be implemented? (And add on the relevant ticket that this patch needs to be removed), this way it will work for all usages and tools & examples will remain the same.
What do you think?

Reply:
I reversed the change. Will be handled in a different PR tracking [LRS-508](https://rsjira.intel.com/browse/LRS-508)